src/core/identifiers/identifiers.ts

```diff
@@ -36,7 +43,18 @@ export type DSNPUserId = string;
  */
 export const isDSNPUserId = (id: unknown): id is DSNPUserId => {
   if (!isString(id)) return false;
-  return id.match(/^dsnp:\/\/0x[0-9A-F]{16}$/i) !== null;
+  return id.match(/^0x[0-9A-F]{1,16}$/i) !== null;
+};
+
+/**
+ * isDSNPUserUri validates a given object as a DSNPUserUri.
+ *
+ * @param uri - The object to validate
+ * @returns True of false depending on whether the string is a valid DSNPUserUri
+ */
+export const isDSNPUserUri = (uri: unknown): uri is DSNPUserUri => {
+  if (!isString(uri)) return false;
+  return uri.match(/^0x[0-9A-F]{1,16}$/i) !== null;
```

Comment:
This is the same regex as `isDSNPUserId` above. I'm pretty sure this one is supposed to start with `dsnp://`.

Reply:
fixed